02-Logging/04-Writing-Formatted-Output.ipynb

```diff
@@ -244,26 +244,26 @@
      "output_type": "stream",
```

Comment:
<p>I am not sure why, but it seems that the <code>Table</code>  writer is adding a newline after each call now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>This is odd, it is writing '\r\n' at the end of each line when writing to a file. That is a Windows line ending. We open the file with the 'x' mode <code>open('log.txt', 'x')</code>.  The docs claim that text mode is default: https://docs.python.org/3/library/functions.html#open</p><p> </p><p>Looking further in the docs, it seems that we can force the correct new line with the newline argument to open.</p>